wgpu-types/src/lib.rs

```diff
@@ -459,14 +459,14 @@ bitflags::bitflags! {
         /// - All
         ///
         /// This is a native-only feature.
-        const VERTEX_WRITABLE_STORAGE = 0x0000_0002_0000_0000;
+        const VERTEX_WRITABLE_STORAGE = 1 << 33;
         /// Enables clear to zero for buffers & images.
         ///
         /// Supported platforms:
         /// - All
         ///
         /// This is a native only feature.
-        const CLEAR_COMMANDS = 0x0000_0400_0000_0000;
+        const CLEAR_COMMANDS = 1 << 42;
```

Comment:
Jump from 33 to 42?

Reply:
I was just copying what the existing flags were, I wasn't sure if there was a reason for the jump or not. If it's native only are these values not tied to the WebGPU spec?

Reply:
Yeah they aren't. I don't know why we would jump here. Likely just due to how difficult maintaining the old way was :)

Reply:
Ok, pushed an update to fix.